models/classes/import/QtiTestImporter.php

```diff
@@ -35,32 +38,33 @@ class QtiTestImporter extends AbstractTestImporter
 
     /**
      * @param File $file
-     * @param \core_kernel_classes_Class $class
+     * @param core_kernel_classes_Class|null $class
      * @param bool $enableMetadataGuardians
      * @param bool $enableValidators
      * @param bool $itemMustExist
      * @param bool $itemMustBeOverwritten
-     * @return \common_report_Report
+     * @param string|null $overwriteTestUri
+     * @param string|null $itemClassUri
+     * @return common_report_Report
      */
     public function import(
         File $file,
-        \core_kernel_classes_Class $class = null,
-        $enableMetadataGuardians = true,
-        $enableValidators = true,
-        $itemMustExist = false,
-        $itemMustBeOverwritten = false,
-        bool $overwriteTest = false,
+        core_kernel_classes_Class $class = null,
+        bool $enableMetadataGuardians = true,
+        bool $enableValidators = true,
+        bool $itemMustExist = false,
+        bool $itemMustBeOverwritten = false,
+        ?string $overwriteTestUri = null,
```

Comment:
Ok, so now we must always know the testUri to be able to use this method and API, which means, we have a breaking change? I am not sure PISA is the only one use it. Would it be possible to no introduce this breaking change and make it backward compatible?

Reply:
As I see, this parameter was added by @yaraslau-kavaliou for PISA. So I think, that no one is using it. @yaraslau-kavaliou can you confirm that?

But sure, for safety I can keep the old one and add this as a new one.